### PR TITLE
analytical Stokes benchmark for 3D hollow sphere

### PR DESCRIPTION
**Pull request by cedrict**
I present here a new family of analytical flow solutions to the incompressible Stokes equation in a spherical shell. The velocity is tangential to both inner and outer boundaries, the viscosity is radial and of power-law type, and the solution has been designed so that the expressions for velocity, pressure, and body force are simple polynomials and therefore simple to implement in (geodynamics) codes.

### CODE REVIEW

File: benchmarks/hollow_sphere/hollow_sphere.cc [hunk 1]
```diff
@@ -0,0 +1,702 @@
...
+#include <deal.II/numerics/error_estimator.h>
+#include <deal.II/numerics/vector_tools.h>
+
+
+
+namespace aspect
+{
+  /**
+   * This is the "Hollowsphere" benchmark defined in the following paper:
+   */
```
Comment (bangerth) [inline benchmarks/hollow_sphere/hollow_sphere.cc]: :-)

Comment (cedrict) [inline benchmarks/hollow_sphere/hollow_sphere.cc]: yes, the low resolution case (768 elements) runs quickly and yes I will add a page in the manual and a test.

Comment (bangerth) [inline benchmarks/hollow_sphere/hollow_sphere.cc]: And the reference missing here? :-)

Comment (cedrict) [inline benchmarks/hollow_sphere/hollow_sphere.cc]: just did it :)

File: benchmarks/hollow_sphere/hollow_sphere.cc [hunk 2]
```diff
@@ -0,0 +1,704 @@
...
+
+        return hr*cos(theta);
+      }
+
+
+      /**
+       * The exact solution for the Hollowsphere benchmark.
+       */
+      template <int dim>
+      class FunctionHollowsphere : public Function<dim>
```
Comment (cedrict) [inline benchmarks/hollow_sphere/hollow_sphere.cc]: I am not sure I understand what is wrong here. my .cc file started as burstedde.cc and I replace 'Burstedde' everywhere by 'Hollowsphere'

Comment (bangerth) [inline benchmarks/hollow_sphere/hollow_sphere.cc]: I think he was suggesting to call the class `HollowSphere` (with a capital `S` in the middle) to match our usual naming convention of using CamelCase for ClassNames.

Comment (bangerth) [inline benchmarks/hollow_sphere/hollow_sphere.cc]: But then you would have to do that everywhere else in this file as well...

Comment (cedrict) [inline benchmarks/hollow_sphere/hollow_sphere.cc]: ok, sure, could do that

**Comment by bangerth** [comment]
Please squash and rebase to force the tester to try again.

**Pull request merged**